airflow/providers/google/cloud/hooks/datafusion.py

```diff
@@ -339,6 +341,8 @@ def delete_pipeline(
         response = self._cdap_request(url=url, method="DELETE", body=None)
         if response.status != 200:
             raise AirflowException(f"Deleting a pipeline failed with code {response.status}")
+        if response.data is None:
+            raise AirflowException("Deleting a pipeline failed because of empty data in received response")
 
```

Comment:
It's fine in general though we have a lot of repeated code - ie get/delete/create operations have all != 200 and None checks.

Reply:
Yes, i will try to use another approach to decrease duplication in the code
Thanks!